ProjFS.Mac/PrjFSLibTests/JsonWriterTests.mm

```diff
@@ -0,0 +1,209 @@
+#include "../PrjFSLib/Json/JsonWriter.hpp"
+#include <utility>
+#include <vector>
+#import <XCTest/XCTest.h>
+
+using std::make_pair;
+using std::pair;
+using std::string;
+using std::vector;
+
+@interface JsonWriterTests : XCTestCase
```

Comment:
@jeschu1 - FYI these tests get us 100% coverage of `JsonWriter`

Reply:
@wilbaker big hi-fives on the unit tests.  You definitely cover some great cases there.